app/views/devise/confirmations/new.html.haml

```diff
@@ -0,0 +1,16 @@
+= content_for :page_title, flush: true do
+  = 'Resend confirmation instructions'
+
+= render partial: 'layouts/header', locals: {page_heading: 'Resend confirmation instructions'}
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }, builder: GOVUKDesignSystemFormBuilder::FormBuilder) do |f|
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698/files#r565935858)